mitodl/open-discussions
# add to README and docstring

#### What's this PR do?
This is a really small PR. It does two simple things:
- Renames previous availability values (user front-facing) to new ones.
- Adds to README about how to search course catalog elasticsearch index.

#### What are the relevant tickets?
closes #2067

#### How should this be manually tested?
Manually run migrations and see if all availability values have been renamed.
Follow the README instructions.
